plugins/kubernetes/app/models/kuber_deploy_service.rb

```diff
@@ -46,6 +47,20 @@ def create_services!
     end
   end
 
+  def delete_replication_controllers!
+    kuber_release.release_docs.each do |release_doc|
+      log 'deleting existing ReplicationControllers', role: release_doc.kubernetes_role.name
+      labels = {project: release_group.build.project_name, role: release_doc.kubernetes_role.label_name}
```

Comment:
Style nitpick: for single-line hashes, leave a space around the curly braces.

e.g. `{ foo: bar }`, not `{foo: bar}`